tests/data/test_configs/CLI_test_config.yml

```diff
@@ -0,0 +1,8 @@
+# This config is used to change the fileview for certain CLI tests
+asset_store:
+  synapse:
+    config: "../../../.synapseConfig"
```

Comment:
Is this synapse config actually used?

Reply:
Hmm I'm assuming that there's somewhere in the CI pipeline that fills in the credentials?